core/mutator/entry/mutator.go

```diff
@@ -87,6 +87,10 @@ func ReduceFn(leaves []*pb.EntryUpdate, msgs []*pb.EntryUpdate,
 	// Filter for mutations that are valid.
 	newEntries := make([]*pb.EntryUpdate, 0, len(msgs))
 	for i, msg := range msgs {
+		// Skip if there is no change in data.
+		if len(leaves) > 0 && bytes.Equal(leaves[0].GetCommitted().GetData(), msg.GetCommitted().GetData()) {
```

Comment:
This worries me that it will skip updates that intentionally keep the data the same, but modify, e.g. the 'authorized_keyset' property. Do we need a deeper equality check than just the data?

Reply:
I think an additional check of the "authorized_keyset" should be sufficient. Agree?

Reply:
@gdbelvin - what are your thoughts on this? What aspects of a KT record are important for equality?